lib/engine/ability/choose_ability.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require_relative 'base'
+
+module Engine
+  module Ability
+    class ChooseAbility < Base
```

Comment:
why is this needed

Reply:
@outsidePasser you didn't answer my question

Reply:
I asked him to create a choose ability class instead of hardcoding into the game class like 1822 did, so the choices can be encapsulated into the company.